### PR TITLE
Markdown conversion for `ru` - Cleanup - MathML section

### PR DESCRIPTION
**Pull request by SphinxKnight**
No report here as no error were left.

### CODE REVIEW

File: files/ru/web/mathml/element/math/index.html [hunk 1]
```diff
@@ -5,120 +5,118 @@
 ---
 <p>{{MathMLRef()}}</p>
 
-<p><span>Элементом верхнего уровня в MathML является тэг <code>&lt;math&gt;</code> .</span> <span style=""> Каждый допустимый экземпляр MathML должен быть внутри этого контейнера.</span> <span> Он не допускает вложений, но внутри может быть произвольное число других дочерних элементов.</span></p>
+<p>Элементом верхнего уровня в MathML является тэг <code>&lt;math&gt;</code> .  Каждый допустимый экземпляр MathML должен быть внутри этого контейнера.  Он не допускает вложений, но внутри может быть произвольное число других дочерних элементов.</p>
```
Comment (sashasushko) [inline files/ru/web/mathml/element/math/index.html:8]:
```suggestion
<p>Элементом верхнего уровня в MathML является тэг <code>&lt;math&gt;</code>. Каждый допустимый экземпляр MathML должен быть внутри этого контейнера. Он не допускает вложений, но внутри может быть произвольное число других дочерних элементов.</p>
```

File: files/ru/web/mathml/element/math/index.html [hunk 2]
```diff
@@ -5,120 +5,118 @@
...
+ <dd>Указывает направление формулы: <code>ltr</code> - слева направо или <code>rtl</code> - справа налево.</dd>
  <dt id="attr-href">ref</dt>
- <dd><span>Используется для установки гиперссылки на указанный URI.</span></dd>
- <dt id="attr-mathbackground"><span>mathbackground</span></dt>
- <dd><span>Цвет фона.</span> <span> Вы можете использовать <code>#rgb</code> , <code>#rrggbb</code> и <a href="https://developer.mozilla.org/en-US/docs/CSS/color_value#Color_Keywords">названия цветов HTML</a> .</span></dd>
- <dt id="attr-mathcolor"><span>mathcolor</span></dt>
- <dd><span>Цвет текста.</span> <span> Вы можете использовать <code>#rgb</code> , <code>#rrggbb</code> и <a href="https://developer.mozilla.org/en-US/docs/CSS/color_value#Color_Keywords">названия цветов HTML</a> .</span></dd>
+ <dd>Используется для установки гиперссылки на указанный URI.</dd>
+ <dt id="attr-mathbackground">mathbackground</dt>
+ <dd>Цвет фона.  Вы можете использовать <code>#rgb</code> , <code>#rrggbb</code> и <a href="https://developer.mozilla.org/en-US/docs/CSS/color_value#Color_Keywords">названия цветов HTML</a> .</dd>
```
Comment (sashasushko) [inline files/ru/web/mathml/element/math/index.html:22]:
```suggestion
 <dd>Цвет фона. Вы можете использовать <code>#rgb</code>, <code>#rrggbb</code> и <a href="https://developer.mozilla.org/en-US/docs/CSS/color_value#Color_Keywords">названия цветов HTML</a>.</dd>
```

File: files/ru/web/mathml/element/math/index.html [hunk 3]
```diff
@@ -5,120 +5,118 @@
...
+ <dt id="attr-mathcolor">mathcolor</dt>
+ <dd>Цвет текста.  Вы можете использовать <code>#rgb</code> , <code>#rrggbb</code> и <a href="https://developer.mozilla.org/en-US/docs/CSS/color_value#Color_Keywords">названия цветов HTML</a> .</dd>
```
Comment (sashasushko) [inline files/ru/web/mathml/element/math/index.html:24]:
```suggestion
 <dd>Цвет текста. Вы можете использовать <code>#rgb</code>, <code>#rrggbb</code> и <a href="https://developer.mozilla.org/en-US/docs/CSS/color_value#Color_Keywords">названия цветов HTML</a>.</dd>
```

File: files/ru/web/mathml/element/math/index.html [hunk 4]
```diff
@@ -5,120 +5,118 @@
...
+ <dd>Этот атрибут определяет способ вывода. Возможные значения:
  <ul>
-  <li><span><code>block</code> - означает, что этот элемент будет отображаться за пределами текущего диапазона текста, как блока, который может быть расположен в любом месте без изменения смысла текста;</span></li>
-  <li><span><code>inline</code> -означает, что этот элемент будет отображаться внутри текущего диапазона текста, и не могут быть перемещены из неё без изменения значения этого текста.</span></li>
+  <li><code>block</code> - означает, что этот элемент будет отображаться за пределами текущего диапазона текста, как блока, который может быть расположен в любом месте без изменения смысла текста;</li>
+  <li><code>inline</code> -означает, что этот элемент будет отображаться внутри текущего диапазона текста, и не могут быть перемещены из неё без изменения значения этого текста.</li>
  </ul>
 
- <p><span>Значение по умолчанию <code>inline</code> .</span></p>
+ <p>Значение по умолчанию <code>inline</code> .</p>
```
Comment (sashasushko) [inline files/ru/web/mathml/element/math/index.html:32]:
```suggestion
 <p>Значение по умолчанию <code>inline</code>.</p>
```

File: files/ru/web/mathml/element/math/index.html [hunk 5]
```diff
@@ -5,120 +5,118 @@
...
 
 <p> </p>
 
-<p><span><strong>Примечания:</strong> XHTML документы с MathML должны быть поданы как <code>application/xhtml+xml</code>.</span> <span> Вы можете легко добиться этого, добавив <code>.xhtml</code> расширение для локальных файлов.</span> <span> Для серверов Apache вы можете <a class="external external-icon" href="http://httpd.apache.org/docs/2.2/mod/mod_mime.html#addtype">настроить <code>.htaccess</code> файл</a> для этого расширения на правильный тип MIME.</span> <span> Поскольку мы сохранили наш MathML в виде XML-документа, необходимо быть уверенным в правильно оформленном XML-документе.</span></p>
+<p><strong>Примечания:</strong> XHTML документы с MathML должны быть поданы как <code>application/xhtml+xml</code>.  Вы можете легко добиться этого, добавив <code>.xhtml</code> расширение для локальных файлов.  Для серверов Apache вы можете <a class="external external-icon" href="http://httpd.apache.org/docs/2.2/mod/mod_mime.html#addtype">настроить <code>.htaccess</code> файл</a> для этого расширения на правильный тип MIME.  Поскольку мы сохранили наш MathML в виде XML-документа, необходимо быть уверенным в правильно оформленном XML-документе.</p>
```
Comment (sashasushko) [inline files/ru/web/mathml/element/math/index.html:119]:
```suggestion
<p><strong>Примечания:</strong> XHTML документы с MathML должны быть поданы как <code>application/xhtml+xml</code>. Вы можете легко добиться этого, добавив <code>.xhtml</code> расширение для локальных файлов. Для серверов Apache вы можете <a class="external external-icon" href="http://httpd.apache.org/docs/2.2/mod/mod_mime.html#addtype">настроить <code>.htaccess</code> файл</a> для этого расширения на правильный тип MIME. Поскольку мы сохранили наш MathML в виде XML-документа, необходимо быть уверенным в правильно оформленном XML-документе.</p>
```

**Review by sashasushko — APPROVED** [review]

**Comment by SphinxKnight** [comment]
Thanks @sashasushko :bow:

**Pull request merged**